frontend/helpers.js

```diff
@@ -31,16 +31,18 @@ export const getEmosproForUid = uid => ({
  * Returns the data for the ec_Event of a product or variant.
  * @param {Object} [baseProduct] The base (parent) product.
  * @param {Object} [variant] The selected variant.
+ * @param {boolean} sendParentUid The parent uid will be sent. Even for a variant article.
```

Comment:
jsDoc: please mark this param as optional and add the default value

Reply:
done